docs/features/ui-language.md

```diff
@@ -15,7 +15,7 @@ See the demo of the editor in German:
 
 ## Building the editor using a specific language
 
-Currently, it is only possible to change the UI language at the build stage. A single build of the editor supports only the language which was defined in the [CKEditor 5 webpack plugin](https://www.npmjs.com/package/@ckeditor/ckeditor5-dev-webpack-plugin)'s configuration.
+Currently, it is possible to change the UI language at the build stage and after the build. A single build of the editor supports the language which was defined in the [CKEditor 5 webpack plugin](https://www.npmjs.com/package/@ckeditor/ckeditor5-dev-webpack-plugin)'s configuration. See the whole translation process to see how you can change the language later.
```

Comment:
This guide needs to be pretty much rewritten rather than patched. Since the whole picture changed dramatically  we need to put focus on different things.

1. The current intro can stay.
1. But then we should explain how to initialize the editor from CDN/zip/npm with a different language. This is what most people will want to do. Please also review other guides like `installation.md` and other places where we list files available in a build because that list needs to be updated (and perhaps link to the ui-language.md guide):

   ![image](https://user-images.githubusercontent.com/156149/33320679-57a0f70a-d443-11e7-9603-845d51da6f59.png)

2. Then we can provide a few more details on what other options people have:

   * compiling an optimised editor (single language case),
   * compiling a subset of languages (TBH, I'm not sure what's the use case for that – perhaps speeding up the process and reducing the number of files – but let's mention it anyway),
   * handling multiple entry points scenario (as @ma2ciek noticed that has certain limitations)

3. At the end we can mention that compiling editor translations is only available for Webpack and we will be extending support for other bundlers in the future. If we want to mention those tickets about translations services we should explain why we mention them. Something like – if you'd like to tinker with one of the bundlers you can read about CKEditor 5's translation service here and here.